lib/modules/apostrophe-pages/lib/routes.js

```diff
@@ -292,7 +292,7 @@ module.exports = function(self, options) {
     if (!manager) {
       return next('notfound');
     }
-    return self.find(req, { _id: _id }).ancestors(true).permission('edit-apostrophe-page').toObject(function(err, page) {
+    return self.find(req, { _id: _id }).ancestors(true).permission('view-apostrophe-page').toObject(function(err, page) {
```

Comment:
There should be no change in how we handle the parent page.

The change is that we now have a new join in the schema for picking the source to be copied from. Which defaults to the parent page, but can be browsed to pick another source.